frameworks/core_foundation/views/view/enabled.js

```diff
@@ -204,6 +204,7 @@ SC.View.reopen(
     // Update the display if in a visible state.
     switch (state) {
     case SC.CoreView.ATTACHED_SHOWN:
+    case SC.CoreView.ATTACHED_SHOWN_ANIMATING:
```

Comment:
Clear... did a quick reading up on optimization of switch cases by V8 among others, but couldn't find much against it. My proposal is to add a specific comment here `// fallthrough` to indicate that this is done on purpose.